src/ui/component/tagsDragAndDrop/view.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
```

Comment:
I need to add the drop to delete functionality. Waiting to hear back on this:
https://github.com/atlassian/react-beautiful-dnd/issues/1331